app/code/Magento/LoginAsCustomer/Test/Mftf/Test/AdminLoginAsCustomerLoggingFilterTest.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminLoginAsCustomerLoggingFilterTest">
+        <annotations>
+            <features value="Login as Customer logs"/>
+            <stories value="Filter by date login as customer logs"/>
+            <title value="Filter by date login as customer logs"/>
+            <description value="Filter by date should be from/to"/>
+            <severity value="MAJOR"/>
+            <group value="login_as_customer"/>
+            <testCaseId value="MC-"/>
+        </annotations>
+        <before>
+            <magentoCLI command="config:set {{LoginAsCustomerConfigDataEnabled.path}} 1"
+                        stepKey="enableLoginAsCustomer"/>
+            <magentoCLI command="config:set {{LoginAsCustomerStoreViewLogin.path}} 0"
+                        stepKey="enableLoginAsCustomerAutoDetection"/>
+            <magentoCLI command="cache:flush config" stepKey="flushCacheBeforeTestRun"/>
+            <createData entity="Simple_US_Customer_Assistance_Allowed" stepKey="createFirstCustomer"/>
+            <actionGroup ref="AdminLoginActionGroup" stepKey="loginAsDefaultUser"/>
+        </before>
+        <after>
+            <!--Clean filters -->
+            <click selector="{{AdminLoginAsCustomerLogToolbarSection.resetFilter}}" stepKey="CleanFiltersAfter"/>
```

Comment:
Can we use this action group instead: ClearFiltersAdminDataGridActionGroup?

Reply:
Has been updated